qark/report.py

```diff
@@ -28,21 +28,21 @@ class Report(object):
     # http://python-3-patterns-idioms-test.readthedocs.io/en/latest/Singleton.html#the-singleton
     __instance = None
 
-    def __new__(cls, report_path=None):
+    def __new__(cls, issues=None, report_path=None):
         if Report.__instance is None:
             Report.__instance = object.__new__(cls)
-        Report.__instance.report_path = report_path
+
         return Report.__instance
 
-    def __init__(self, report_path=None):
+    def __init__(self, issues=None, report_path=None):
         """This will give you an instance of a report, with a default report path which is local
         to where QARK is on the file system.
 
         :param report_path: The path to the report directory where all generated report files will be written.
         :type report_path: str or None
 
         """
-        self.issues = []
+        self.issues = issues if issues is not None else []
```

Comment:
Is there a reason you compare explicitly with `None`? 
Cause otherwise this could become `issues if issues else []`

Reply:
Will change